### DISCUSSION TITLE
Wallbash

### DISCUSSION THREAD

**Discussion by dieBakterie** [prasanthrangan/hyprdots#1531] [Q&A] [answered]
Why does the Theme doesn't apply to Discord if wallbashtoggle is used and switched to Theme mode?

**Comment by kRHYME7** [comment]
Not really an issue.  It is an intended approach. I forget why I decided to not theme discord with wallbash if wallbash is disabled. 

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/9e9ff907-ab40-4bf0-93e9-06f30f02b04b)


You can create an issue or a PR to remove this condition

**Reply by dieBakterie** [reply]
Ohh don't know how the scripts work and just looool

**Reply by kRHYME7** [reply]
I added the discord template not knowing if the text is readable, so I added that condition in case discord looks broken.

**Reply by dieBakterie** [reply]
You meant like this
```
for client_css in "${client_list[@]}"; do
    eval client_css="${client_css}"

    if [[ ! -d $(dirname "${client_css}") ]]; then
        cp "${discord_col}" "${client_css}"
    fi
    
    # if [[ "${enableWallDcol}" -gt 0 ]]; then
    #     cp "${discord_col}" "${client_css}"
    # else
    #     > "${client_css}"
    # fi
done
```
?

**Reply by dieBakterie** [reply]
Ohh i fought theme mode is **Theme** mode and not disabled wallbash 🤣

**Reply by kRHYME7** [reply, marked as answer]
```

#// main loop

for client_css in "${client_list[@]}" ; do
    eval client_css="${client_css}"
    if [[ ! -d $(dirname "${client_css}") ]] ; then
        continue
    fi
        cp "${discord_col}" "${client_css}"
done
```
#### or

```
#// main loop

for client_css in "${client_list[@]}" ; do
    eval client_css="${client_css}"
    if [[  -d $(dirname "${client_css}") ]] ; then
        cp "${discord_col}" "${client_css}"
    fi
done
```

**Reply by kRHYME7** [reply]
You can do a PR and I think we can merge it

**Reply by dieBakterie** [reply]
Yeah but in Graphite Mono you can't really See the Text on popouts like when you Hover over a Servers Name and the User Profile edit Button is invisible unless you've your Mouse on him and the remove authenticator App Button too. Propably in some more light Themes too.

**Reply by dieBakterie** [reply]
But i've added opaque for my discord(chaotic aur vesktop-git) package
